### DISCUSSION TITLE
Build Failed Command "npm run build" exited with 1

### DISCUSSION THREAD

**Discussion by danpbellucci** [vercel/vercel#8445] [Help] [answered]
Hello.
I did a minor update on my website but the deploy is not working.

This is what I changed on the website code:
![image](https://user-images.githubusercontent.com/21298197/186057468-de7fbe49-7ca1-4880-b5de-f7bec8a040a4.png)

And this error occurs when I try to deploy.

![image](https://user-images.githubusercontent.com/21298197/186057584-f9dc7406-cd08-4f53-bcb4-5c2175259db0.png)

**Comment by amyegan** [comment, marked as answer]
Hi @danpbellucci! If I understand your solution, it looks like there is a problem with one of your templates. Seems like whatever object you're trying to read `rows` from is missing, so maybe some data isn't coming through or one of the IDs has a typo. I can't say for sure without knowing how your project is supposed to work.

If you can share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I would be happy to take a closer look.

**Reply by danpbellucci** [reply]
Thank you so much for your replay, amyegan! I appreciate.
